src/libutil/kv.c

```diff
@@ -204,7 +204,7 @@ static int kv_put_raw (struct kv *kv, const char *key, enum kv_type type,
 
 int kv_vput (struct kv *kv, const char *key, enum kv_type type, va_list ap)
 {
-    char s[80];
+    char s[317]; // DBL_MAX with 6 digit precision + \0
```

Comment:
Should one more byte be added to the buffer in case of `-DBL_MAX`? What is one more byte among 317 friends?

Reply:
I added -DBL_MAX to the test and some wiggle room to that buffer.